src/core/style/themes.scss

```diff
@@ -39,4 +39,10 @@ html, body {
   &#{$checkedSelector} ._md-icon:after {
     border-color: '{{primary-contrast-0.87}}';
   }
+
+  & .md-indeterminate[disabled] {
```

Comment:
@EladBezalel In response to private: I had a discussion with Thomas where we thought that CSS classes representing the state of a component should probably be public as they are likely the ones to be used most.